src/test/groovy/org/ods/doc/gen/leva/doc/fixture/LevaDocTestValidator.groovy

```diff
@@ -49,6 +59,17 @@ class LevaDocTestValidator {
         } else {
             FileUtils.copyFile(actualDoc(buildId), reportPdfDoc(buildId))
         }
+
+        if (!filesAreEqual) {
```

Comment:
This is the else of the line 59.
Improve the else of the if.

Reply:
We'll do that if previous change is accepted. If not, we'll remove this lines because they are related to the previous ones.